library/packages/src/lib/y2packager/resolvable.rb

```diff
@@ -80,6 +80,17 @@ def initialize(hash)
       from_hash(hash)
     end
 
+    # Backward compatibility method to access resolvable like hash.
+    def [](key)
+      log.info "Calling [] with #{key}. Deprecated. Use method name directly"
```

Comment:
Maybe also log the caller so we can easily find the place where it was called from?

Reply:
good idea.